data/json/items/comestibles.json

```diff
@@ -14659,6 +14659,33 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "fresh_fries",
+        "name" : "French fries",
+        "name_plural" : "French fries",
+        "weight" : 115,
+        "color" : "yellow",
+        "addiction_type" : "none",
+        "spoils_in" : 72,
+        "stim" : 0,
+        "container" : "box_small",
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : -1,
+        "healthy" : -1,
+        "addiction_potential" : 0,
+        "nutrition" : 3,
+        "description" : "Deep fried potatos with a touch of salt.  Crunchy and delicious.",
+        "price" : 120,
+        "material" : "junk",
+        "tool" : "null",
+        "volume" : 1,
+        "charges" : 3,
+        "bashing" : 1,
```

Comment:
That's weird, I don't think that French fried could hurt someone, so bashing : 1 is too much. If it's the same as in initial version then you can fix this as well.